scripts/add_component_skeleton.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+
+import argparse
+from os import chmod, makedirs
+import os.path
+
+check = (
+    "Check jobs are triggered when a pull request is created. Any job can be "
+    "added as a check, the most common use is for lint, unit and basic "
+    "functional tests."
+)
+
+gate = (
+    "Gate jobs are triggered by a comment on a pull request. The change is "
+    "merged automatically if all the tests pass. The gate job ensures that "
+    "changes are tested against the state of the target branch when the code "
+    "is merged, in comparison check jobs test against the state of the branch "
```

Comment:
`The gate job ensures that changes are tested against the state of the target branch when the code is merged. In comparison, check jobs test against the state of the branch when the test is started.`

Does that read slightly better?

Reply:
Maybe. I'm happy to change it.